ui/src/components/filebrowser.vue

```diff
@@ -45,6 +45,15 @@
 import Vue from 'vue'
 
 import mute from '@/components/mute'
+import VueTimeago from 'vue-timeago'
```

Comment:
Can this initialization done in main.js? Or does it have to be done on all components that uses <timeago>?